tests/migration/20_reindex.yml

```diff
@@ -0,0 +1,56 @@
+---
+requires:
+  stack: true
+  serverless: false
+---
+teardown:
+  - do:
+      indices.delete_data_stream:
+        name: test-reindex-datastream
+---
+"Test migrate_reindex, get_migrate_reindex_status, cancel_migrate_reindex":
+  - do:
+      indices.put_index_template:
+        name: test-reindex-template
+        body:
+          index_patterns: [test-reindex-datastream*]
+          template:
+            mappings:
+              properties:
+                '@timestamp':
+                  type: date
+                'foo':
+                  type: keyword
+          data_stream: {}
+
+  - do:
+      indices.create_data_stream:
+        name: test-reindex-datastream
+  - is_true: acknowledged
+
+  - do:
+      indices.migrate_reindex:
+        body: |
+          {
+            "mode": "upgrade",
+            "source": {
+              "index": "test-reindex-datastream"
+            }
+          }
+  - match: { acknowledged: true }
```

Comment:
what's the difference between `match: { acknowledged: true }` and `is_true: acknowledged`? You're using both in this file.

Reply:
They're the same, but I updated it to use `is_true` for consistency :+1: